### PR TITLE
Bug 1270736 - update paths to shared resources in clock app r=albertopq

### PR DESCRIPTION
**Pull request by TitanNano**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d7353b13d0e585e0f10ef2d2ad6830cba6f5bb24

File: dead_apps/clock/index.html [hunk 1]
```diff
@@ -15,48 +15,36 @@
     <link rel="stylesheet" type="text/css" href="style/picker/picker.css">
 
     <!-- Shared styles -->
-    <link rel="stylesheet" type="text/css" href="shared/style/status.css">
-    <link rel="stylesheet" type="text/css" href="shared/style/tabs.css">
+    <link rel="stylesheet" type="text/css" href="style/status.css">
```
Comment (albertopq) [inline dead_apps/clock/index.html]: If we are using the ../shared path in the other files, lets do it in this one too

File: dead_apps/clock/js/panels/alarm/active_alarm.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ define(function(require) {
       window.location.origin + '/onring.html');
 
     // Handle the system's alarm event.
-    navigator.mozSetMessageHandler('alarm', this.onMozAlarm.bind(this));
+    // navigator.mozSetMessageHandler('alarm', this.onMozAlarm.bind(this));
```
Comment (albertopq) [inline dead_apps/clock/js/panels/alarm/active_alarm.js]: Lets create a bug for finding an alternative to Alarms, and add the bug number to this comment

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=42e4c419b116120814b4efba75927c5dc4cc51f1

File: dead_apps/clock/index.html [hunk 3]
```diff
@@ -15,48 +15,36 @@
...
-    <!--
-      The following scripts are lazy loaded but left here to ensure
-      they get copied over from shared during the build
-    -->
-    <!-- <script defer src="/shared/js/async_storage.js"></script> -->
-    <!-- <script defer src="/shared/js/gesture_detector.js"></script> -->
-    <!-- <script defer src="/shared/js/accessibility_helper.js"></script> -->
-    <!-- <script defer src="/shared/js/template.js"></script> -->
+    <link rel="stylesheet" type="text/css" href="../../shared/style/input_areas.css" >
+    <link rel="stylesheet" type="text/css" href="style/buttons.css" >
```
Comment (albertopq) [inline dead_apps/clock/index.html]: Why is this file not pointing to ../../shared? I think input_areas.css and buttons.css files should be remove from this commit too.

File: dead_apps/clock/js/panels/alarm/active_alarm.js (hunk 2 above)
Comment (TitanNano) [inline dead_apps/clock/js/panels/alarm/active_alarm.js]: [Bug 1270736](https://bugzilla.mozilla.org/show_bug.cgi?id=1270736)

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a0e2cfc355f7daf20cab7fff8404f21e8a9808f1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a0e2cfc355f7daf20cab7fff8404f21e8a9808f1

File: dead_apps/clock/js/panels/alarm/active_alarm.js (hunk 2 above)
Comment (albertopq) [inline dead_apps/clock/js/panels/alarm/active_alarm.js]: Oh, I meant to the code comment, but no worries. We can live without that.

**Pull request merged**